angular.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    extends: [
+        'plugin:@angular-eslint/recommended',
+        'plugin:@angular-eslint/recommended--extra',
+        'plugin:@angular-eslint/template/process-inline-templates',
+        './typescript',
+        './typescript-requiring-type-checking'
```

Comment:
For TypeScript, we provide two separate rule sets to give clients the option to run the rules requiring type checking separately. Do we want the same for Angular? Or do the angular rules require type checking anyway?

Reply:
Actually, it's the other way around. From the description, "Disabling the parse options, which should cause rules that require type checking to fail, didn't result in any errors. I assume then that none of the rules require type checking."